example/image-server.py

```diff
@@ -0,0 +1,29 @@
+
+from tornado.ioloop import IOLoop
+from tornado.web import Application, RequestHandler
+
+class MainHandler(RequestHandler):
```

Comment:
Add a comment or rename the class to indicate that this is a fake service.

Reply:
Done